src/st/collision.h

```diff
@@ -198,7 +229,7 @@ void HitDetection(void) {
     s32* scratchpad_2;
     Entity* iterEnt2;
     s32* scratchpad_1;
-    u16* randCompare;
+    u16* dropThresholdAndItem;
```

Comment:
Perhaps just `dropTable`?
That would maintain the naming association with `g_jewelSwordDropTable` and is a little less clunky.